examples/jsm/controls/OrbitControls.js

```diff
@@ -514,6 +591,26 @@ class OrbitControls extends EventDispatcher {
 
 		}
 
+		function updateMouseParameters( event ) {
+
+			const x = event.clientX - scope.domElement.clientLeft;
+			const y = event.clientY - scope.domElement.clientTop;
+			const w = scope.domElement.clientWidth;
+			const h = scope.domElement.clientHeight;
+
+			mouse.x = ( x / w ) * 2 - 1;
+			mouse.y = - ( y / h ) * 2 + 1;
+
+			dollyDirection.set( mouse.x, mouse.y, 1 ).unproject( object ).sub( object.position ).normalize();
+
+		}
+
+		function clampDistance( dist ) {
+
+			return Math.max( scope.minDistance, Math.min( scope.maxDistance, dist ) );
```

Comment:
I think you could reuse the clamp function available in [MathUtils](https://github.com/mrdoob/three.js/blob/517ce63ecc63e675643af6f7c16e4c94092b3e7f/src/math/MathUtils.js#L26)